### PR TITLE
[NETBEANS-54] Module Review libs.nbjshell9

### LINKED ISSUE #130: Cleanup NetBeans license header for module lib.nbjshell9
Review and cleanup the license header in lib.nbjshell9

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in lib.nbjshell9:
```
lib.nbjshell9/build.xml
lib.nbjshell9/src/org/netbeans/lib/nbjshell9/Bundle.properties

```

### PR DESCRIPTION
**Pull request by marcelestial**
Fixes #130

### CODE REVIEW

**Comment by geertjanw** [comment]
Thanks for working on this. The commit message needs to contain the related issue number and should probably be exactly this, instead of "Issue #30":

[NETBEANS-54] Module Review libs.nbjshell9

**Comment by marcelestial** [comment]
Ah, thank you for letting me know.

File: lib.nbjshell9/build.xml [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```
Comment (emilianbold) [inline lib.nbjshell9/build.xml]: This header needs some newlines here. See some other XML files in the repository.

Better still, don't add a license header for build.xml

They are a "Problems to be solved centrally" (see https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review ) and we'll have a script update them all at once in the future.

File: lib.nbjshell9/src/org/netbeans/lib/nbjshell9/Bundle.properties [hunk 2]
```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```
Comment (emilianbold) [inline lib.nbjshell9/src/org/netbeans/lib/nbjshell9/Bundle.properties:9]: This one is OK.

**Review by emilianbold — CHANGES_REQUESTED** [review]

**Review by emilianbold — APPROVED** [review]

**Pull request merged**